ingestion/functions/parsing/ch_zurich/zurich_test.py

```diff
@@ -0,0 +1,57 @@
+import csv
+import json
+import os
+import tempfile
+import unittest
+from datetime import date
+
+import pytest
+
+from ch_zurich import zurich
+
+_SOURCE_ID = "abc123"
+_SOURCE_URL = "https://gd.zh.ch/internet/gesundheitsdirektion/de/themen/coronavirus.html"
+_PARSED_CASE = (
+    {
+        "caseReference": {
+            "sourceId": _SOURCE_ID,
+            "sourceUrl": _SOURCE_URL
+        },
+        "location": {
+            "country": "Switzerland",
+            "administrativeAreaLevel1": "Zurich",
+            "geoResolution": "Admin1",
+            "name": "Zurich canton",
+            "geometry": {
+                "longitude": "8.651071",
+                "latitude": "47.42568"
+            }
+        },
+        "events": [
+            {
+                "name": "confirmed",
+                "dateRange":
+                        {
+                            "start": "02/27/2020Z",
+                            "end": "02/27/2020Z"
+                        }
+            }
+        ],
+        "demographics": {
+            "ageRange": {
+                "start": 33,
+                "end": 33
+            },
+            "gender": "Female"
+        },
+    })
+
+
+class CHZurichTest(unittest.TestCase):
+    def test_parse(self):
+        self.maxDiff = 5000
```

Comment:
Unused (or is there :mage: at play)?

Reply:
when there was a diff in the output, pytest was complaining that it was larger than the 1500 chars that's the default, I increased it so that if a diff comes up it is shown.

Reply:
Gotcha. Does `Testcase.maxDiff` work (in the class, above this method)? Just for the sake of clarity.

Reply:
not sure, the error message said to set self.maxDiff so that's what I did, I think I'd stick to that (considering other test cases might not need that big diff num).